2089_find_target_indices.py

```diff
@@ -0,0 +1,15 @@
+from typing import List
+
+
+class Solution:
```

Comment:
You can solve this problem by a more efficient solution.

Reply:
I made another one as a file "2089_find_target_indeices2.py"
Please check.